easybuild/easyconfigs/j/Java/Java-11.0.6-ppc64le.eb

```
@@ -0,0 +1,15 @@
+name = 'Java'
+version = '11.0.6'
+versionsuffix = '-ppc64le'
+
+homepage = 'https://openjdk.java.net'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/SAP/SapMachine/releases/download/sapmachine-%(version)s%2B4']
```

11.0.6 was release, so:
```suggestion
source_urls = ['https://github.com/SAP/SapMachine/releases/download/sapmachine-%(version)s']
```

---

changed this and the sources line.